getting_started/intel_debug/introduction.rst

```diff
@@ -137,4 +137,36 @@ drivers and should work when changing the ``dsp_driver`` parameter.
 The selection of firmware, topology, and UCM files is based on platform
 capabilities, codec names, and DMI options. While the SOF team and the
 community try to cover all possible cases, errors will happen when the
-wrong file is selected at any of the three layers.
\ No newline at end of file
+wrong file is selected at any of the three layers.
+
+4. Chromebooks and SOF
+----------------------
+
+As stated above, Chromebook firmware is as of 2021 configured with the
```

Comment:
this is more circa 2019/2020

Reply:
I meant "up to today 2021". As a native speaker could you help me rephrase?

Reply:
Ah if you meaning as  "at the time of writing this" then I think this is good. We have 0 intent to change this, so I think its safe to write it as a "beginning in" style fashion to convey when the switch happened